verilog/analysis/json_diagnostics_test.cc

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "verilog/analysis/json_diagnostics.h"
+
+#include "gtest/gtest.h"
+#include "absl/memory/memory.h"
+#include "absl/strings/string_view.h"
+#include "verilog/analysis/verilog_analyzer.h"
+#include "common/util/logging.h"
+
+#include "json/json.h"
+
+namespace verilog {
+namespace {
+
+static void CheckJsonErrorItem(const Json::Value& json, const char* phase, const char* text) {
+  EXPECT_TRUE(json["column"].isIntegral());
+  EXPECT_TRUE(json["line"].isIntegral());
+  EXPECT_TRUE(json["phase"].isString());
```

Comment:
`ASSERT_TRUE` if failing this should prevent the next line from executing.

Same with `ASSERT_TRUE(json["text"].isString());`